test/.env

```diff
@@ -1,4 +1,5 @@
 INDY_NETWORK_NAME=idunion_local_test
+IMAGE_NAME=ghcr.io/idunion/indy-node-container/indy_node:buster
```

Comment:
Why do we need to reference the `buster` image here?

Reply:
I just wanted to have a sane default.